dss/api/files.py

```diff
@@ -67,6 +75,17 @@ def get_helper(uuid: str, replica: Replica, version: str=None):
     ))
 
     if request.method == "GET":
+        '''
```

Comment:
Why single-quote rather than double-quote?  Everywhere else in the repo uses double-quote.

Reply:
fixed